manifests/repo/debian.pp

```diff
@@ -62,8 +62,8 @@
     # both repositories are required to work correctly
     # See: http://www.dotdeb.org/instructions/
     if $release == 'wheezy-php56' {
-      ::apt::source { 'dotdeb-wheezy':
-        location => $location,
+      ::apt::source { 'php-wheezy':
+        location => 'http://packages.dotdeb.org',
```

Comment:
we can use $location here, right?

Reply:
this is the question, if we want this module to auto manage the repos, we have to add hardcoded locations. like 
- if `stretch` and `php version` = `7.1` or `5.6` auto use sury
- if `wheezy` ... use dotdeb or sury (or sury for all, we have to build a matrix what repo should automatically be used

some kind of if location is undef -> use auto mode, else do not use any hardcoded repo